docs/api/target.rst

```diff
@@ -355,3 +355,62 @@ Internally created variables (NAME = "foo"):
 
 Internal variable names are prefixed with ``_`` to avoid collision with input parameters.
 
+.. _blt_import_library:
+
+blt_import_library
+~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: cmake
+
+    blt_import_library( NAME                     <libname>
+                        DEPENDS_ON               [dep1 [dep2 ...]]
+                        INCLUDES                 [include1 [include2 ...]]
+                        TREAT_INCLUDES_AS_SYSTEM [ON|OFF]
+                        FORTRAN_MODULES          [path1 [path2 ..]]
+                        LIBRARIES                [lib1 [lib2 ...]]
+                        COMPILE_FLAGS            [flag1 [flag2 ..]]
+                        LINK_FLAGS               [flag1 [flag2 ..]]
+                        DEFINES                  [def1 [def2 ...]] )
+
+Creates a CMake library target from a list of existing library files or targets.
```

Comment:
>  Creates a CMake library target from \*a list of existing library files or targets. \*

Please clarify the end of the above sentence.

The following might be a bit better, but is still not perfect:
> Creates a CMake target from build artifacts and system files generated outside of this build system.